### PR TITLE
Reimplemented Qbs configurations according Qt-Qbs-Application latest …

### PR DESCRIPTION
**Pull request by AndreyShelest**

### CODE REVIEW

**Comment by approximator** [comment]
Build error `ERROR: /fgap/src/FlightController.qbs:21:8 import QmlTools not found`: 
https://travis-ci.org/approximator/FlightGear_Autopilot/jobs/133837146#L256

Qt-Qbs-Application submodule is at correct commit: 
https://travis-ci.org/approximator/FlightGear_Autopilot/jobs/133837146#L169

File: FgAutopilot.qbs [hunk 1]
```diff
@@ -21,10 +21,14 @@ import qbs.FileInfo
 
 Project {
     name: "fgap_project"
+    minimumQbsVersion: "1.5"
```
Comment (approximator) [inline FgAutopilot.qbs]: Do we really need 1.5?

Comment (AndreyShelest) [inline FgAutopilot.qbs]: I am not shure.  I change qbs version to 1.4 and it is  needed to check build

**Comment by approximator** [comment]
Discard this due to #22

**Pull request closed without merging**